cypress/unit/store-getter-getCategoricalJsonOutput.cy.js

```diff
@@ -0,0 +1,94 @@
+import { getters } from "~/store";
+
+const category = "category1";
+const columnName = "column1";
+let store = {
+
+    getters: getters,
+
+    state: {
+
+        appSetting: { termURLPrefix: "prefix" },
+        categoricalOptions: {
+
+            category1: [
+
+                { label: "annotatedValue1", identifier: "https://example.org/option_1"},
+                { label: "annotatedValue2", identifier: "https://example.org/option_2"}
+            ]
+        },
+        columnToCategoryMap: { column1: "category1" },
+        dataDictionary: {
+
+            annotated: {
+
+                column1: {
+
+                    valueMap: { "rawValue1": "annotatedValue1" }
+                }
+            }
+        }
+    }
+};
+
+describe("getCategoricalJsonOutput", () => {
+
+    it("Make sure categorical json output is schema compliant", () => {
+
+        // Act - Get formatted json output data for discrete value column
+        const output = store.getters.getCategoricalJsonOutput(store.state)(columnName);
+
+        // Assert - Current annotated data dictionary schema compliance
+
+        // Parent level
+
+        expect(output).to.have.property("Annotations");
+
+        // Child level
+
+        expect(output.Annotations).to.have.property("IsAbout");
+        expect(output.Annotations).to.have.property("Levels");
+
+        // Grandchild level
+        expect(output.Annotations.IsAbout).to.have.property("Label");
+        expect(output.Annotations.IsAbout).to.have.property("TermURL");
+        Object.keys(output.Annotations.Levels).forEach(rawValue => {
+
+            expect(output.Annotations.Levels[rawValue]).to.have.property("Label");
+            expect(output.Annotations.Levels[rawValue]).to.have.property("TermURL");
+        });
+    });
+
+    it("Make sure 'IsAbout' fields are properly formatted", () => {
+
+        // Act - Get formatted json output data for discrete value column
+        const output = store.getters.getCategoricalJsonOutput(store.state)(columnName);
+
+        // Assert - Label in this column's 'IsAbout' section to be its assigned category
+        expect(output.Annotations.IsAbout.Label).to.equal(category);
+
+        // Assert - TermURL in this column's 'IsAbout' section is properly formatted
+        // with prefix, colon, and capitalized term
+        expect(output.Annotations.IsAbout.TermURL).to.equal(
```

Comment:
I think it's better to be explicit in this case i.e. use `prefix:Category1` instead of the code used inside the getter for formatting the `TermURL` because if the code is not doing what its supposed to be doing the test won't pick up on it.